images/openshift-ci/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This Dockerfile is used by openshift CI
```

Comment:
Can this be named `Containerfile`?

Reply:
Any particular reason? This remains consistent with: https://github.com/code-ready/crc/tree/master/images

Reply:
We could rename the crc one if this works :) Hopefully OpenShift CI is not using `docker` but `podman`, so the `Dockerfile` name is misleading. But it's just a personal preference to see how far I can go without having `docker` on my machine.

Reply:
Yes, I can rename it but ci-operator (used for openshift-ci) have `dockerfile_path` which required Dockerfile path we can try to change it but all the project under openshift org use `Dockerfile` so I just used the same, we can try to rename it and if CI fails then change it back to Dockerfile (but that would be a separate PR since we need this in to test) ?

Reply:
Not worth spending time on it :)